scripts/k8s/setup-host.sh

```diff
@@ -28,11 +31,21 @@ install_yum_packages() {
     yum group install -y "Development Tools"
 }
 
+install_criu_ubuntu_2004() {
```

Comment:
this says 2004, but what you download is for 22.04

Reply:
facts